src/main/pages/che-7/end-user-guide/proc_configuring-the-workspace-and-adding-tooling.adoc

```diff
@@ -18,6 +18,12 @@ Che 7 plug-ins replace Che 6 installers. The following table lists the Che 7 plu
 | org.eclipse.che.ls.java  | Language Support for Java
 |===
 
+.Prerequisites
+
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{site-baseurl}che-7/che-quick-starts/[Che quick-starts].
```

Comment:
Same old, same old... See the other comments and suggestions about these prereqs.